src/cfi.rs

```diff
@@ -233,18 +387,23 @@ impl<'input, Endian> FrameDescriptionEntry<'input, Endian>
     #[allow(dead_code)]
```

Comment:
I imagine if/when we add an `fde_from_offset` method, this will no longer be dead code. However, I need to read the standard in more depth to see if we even want that method. If not, I'll port all tests over to using `PartialFrameDescriptionEntry`.